.github/workflows/ci.yml

```diff
@@ -0,0 +1,124 @@
+name: CI
+
+on:
+  pull_request:
```

Comment:
Doesn't this mean it won't run on pushes to master? In any case, this can be simplified to `on: pull_request`